rules/avoid-repeated-emojis/rule.md

```diff
@@ -0,0 +1,63 @@
+---
+type: rule
+title: DRY - Do you avoid repeated emojis in lists?
+uri: avoid-repeated-emojis
+authors:
+  - title: Jack Pettit
+    url: https://www.ssw.com.au/people/jack-pettit
+related:
+  - use-emojis
+created: 2023-02-28T00:28:10.330Z
+guid: 1acd6766-d31f-493f-b6f1-3b0b2ff2dc4e
+---
+Emojis are a lot easier to consume when a lot of information is present. But when repeated excessively, they can become a hindrance rather than a help. This helps to make it more visually interesting and easier to scan quickly. In this way, emojis can enhance the communication experience, rather than detracting from it.
+
+<!--endintro-->
+
```

Comment:
Hey @JackDevAU 

1. Can you add a ::: info box here with a short defintion of DRY? E.g. https://www.digitalocean.com/community/tutorials/what-is-dry-development

> D_RY, which stands for ‘don’t repeat yourself,’ is a principle of software development that aims at reducing the repetition of patterns and code duplication in favor of abstractions and avoiding redundancy._